great-idea/index.css

```
@@ -0,0 +1,145 @@
+
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 2px;
+    margin-bottom: 15px;
+}
+
+
+.container {width: 880px;
+	margin: 0 auto; 
+	
+	}
+	header{
+		display: flex;
+		flex-direction: row;
+		
+		
+	}
+	header nav{
+	display: flex;
+	width: 625px;
+	 margin-top: 65px;
+	 justify-content: space-between;
+	 
+
```

try to align everything soo it look neater you can use prettier extention it helps out to make your code look cleaner
```suggestion
.container {width: 880px;
	margin: 0 auto; 
	
	}
	header{
		display: flex;
		flex-direction: row;
		
		
	}
	header nav{
	display: flex;
	width: 625px;
	 margin-top: 65px;
	 justify-content: space-between;
	 

```